pkg/examples/example.go

```diff
@@ -105,8 +105,11 @@ func (eg *Generator) StoreExamples() error { // nolint:gocyclo
 				}
 			}
 		}
+
+		newBuff := bytes.TrimSuffix(buff.Bytes(), []byte("\n---\n\n"))
+
 		// no sensitive info in the example manifest
-		if err := ioutil.WriteFile(pm.ManifestPath, buff.Bytes(), 0600); err != nil {
+		if err := ioutil.WriteFile(pm.ManifestPath, newBuff, 0600); err != nil {
```

Comment:
Could you please replace the deprecated `ioutil.WriteFile` with `os.WriteFile`?

Reply:
I have changed and tested in `turkenf/provider-azuread`:
<img width="1517" alt="Screenshot 2023-07-27 at 23 06 18" src="https://github.com/upbound/upjet/assets/103541666/de0eafb0-3c4c-4963-a25b-baca1df4fb08">